SECP-service/src/main/resources/assets/app/js/home/home.js

```diff
@@ -0,0 +1,18 @@
+(function($) {
```

Comment:
I think this can go in the home-controller.js :)

Reply:
true